app/models/requests/requestable.rb

```diff
@@ -127,7 +127,7 @@ def in_process?
 
     def on_order?
       return false unless item? && !scsb?
-      item[:status_label] == 'Acquisition'
+      ['Acquisition', 'Hold Shelf'].include? item[:status_label]
```

Comment:
Hold Shelf is not an on order related status. We don't want them to be treated like items that have the "Acquisition" status.

Reply:
The spec blocks that I think are in play here https://github.com/pulibrary/requests/blob/24aa750c5fbb27939918a63a7549435eb4788db9/spec/models/requests/requestable_spec.rb#L803 relate to "Pending Orders" in Voyager. At least as I understand all these are now collapsed into the "Acqusition" status in Voyager. "Hold Shelf" would be the status that Alma would provide for items covered by this spec block: https://github.com/pulibrary/requests/blob/24aa750c5fbb27939918a63a7549435eb4788db9/spec/models/requests/requestable_spec.rb#L201.

Reply:
@kevinreiss My latest update should fix this.